src/backends/torch/torchlib.h

```diff
@@ -142,6 +142,8 @@ namespace dd
                                 the file where the weights are stored */
     unsigned int _nclasses = 0;
 
+    std::shared_ptr<spdlog::logger> _logger; /**< mllib logger. */
```

Comment:
This is already defined in `MLLib`, I don't think we should have it again here ?

Reply:
if one want to log directly from torchmodule, I do not see any other way to do

Reply:
If torchmodule is uniquely associated to an instance of torchlib, I'd suggest that the logger from that torchlib object is used, no ?

Reply:
torchmodule does not have access to its container , so we give it the pointer to the logger so that it can use the same logger as its torchlib

Reply:
ok

Reply:
ok